src/unix/tty.c

```diff
@@ -135,9 +138,6 @@ int uv_tty_init(uv_loop_t* loop, uv_tty_t* tty, int fd, int readable) {
   else
     flags |= UV_STREAM_WRITABLE;
 
-  if (!(flags & UV_STREAM_BLOCKING))
```

Comment:
Remember we cannot add some fds to kqueue, you wrote the magic bits! :-)
On May 19, 2016 19:18, "Fedor Indutny" <notifications@github.com> wrote:

> In src/unix/tty.c
> <https://github.com/libuv/libuv/pull/879#discussion_r63928592>:
>
> > @@ -135,9 +138,6 @@ int uv_tty_init(uv_loop_t* loop, uv_tty_t* tty, int fd, int readable) {
> >    else
> >      flags |= UV_STREAM_WRITABLE;
> >
> > -  if (!(flags & UV_STREAM_BLOCKING))
>
> Both select trick fd and original fd set to non-blocking mode?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/libuv/libuv/pull/879/files/2e09fee89d81977a74262c48e533dd47120c213e#r63928592>
>